test/jdk/java/nio/channels/FileChannel/LoopingTruncate.java

```diff
@@ -48,7 +48,11 @@ public class LoopingTruncate {
     static long TIMEOUT = adjustTimeout(20_000);
 
     public static void main(String[] args) throws Throwable {
-        Path path = Files.createTempFile("LoopingTruncate.tmp", null);
+        Path path = Files.createTempFile(
+                // Intentionally opting out from the default `java.io.tmpdir`.
```

Comment:
A comment within the parameter list seems a bit weird as does the indentation.

Reply:
I chose this style since the comment applies to a single argument and is only relevant within `createTempFile()`. Reverted to the old style in 909572e56d87d6a1411c2dba23cfd4c75eaa52ae.